apps/userauth/views.py

```diff
@@ -80,56 +81,59 @@ def post(self, request: WSGIRequest) -> Union[HttpResponse, HttpResponseRedirect
         form = CustomUserAddDataForm(request.POST)
         if current_user.year_of_birth is not None or current_user.post_code is not None:
             return HttpResponse(
-                "You cannot change these values yourself once they are set. Instead, make a request to the administrators via the profile edit page."
+                "You cannot change these values yourself once they are set. "
+                "Instead, make a request to the administrators via the profile edit page."
             )
-        else:
-            if form.is_valid():
-                form.full_clean()
-                current_user.display_name = str(form.cleaned_data.get("display_name"))
-                current_user.year_of_birth = int(form.cleaned_data.get("year_of_birth"))
 
-                current_user.post_code = PostCode.objects.get_or_create(
-                    code=filter_postcode(form.cleaned_data.get("post_code"))
-                )[0]
+        form = CustomUserAddDataForm(request.POST)
+        if not form.is_valid():
+            print("Missing fields? Request data: ", request.POST)
```

Comment:
Debugging statement? ... otherwise should use proper django logger...

Reply:
Good point! Noted for a future improvement.